bl/files/files.go

```diff
@@ -13,24 +15,39 @@ type UnknownStruct map[string]interface{}
 func ExtractFilesConfigurations(paths []string, concurrency int) (chan *extractor.FileConfigurations, chan *extractor.InvalidFile) {
 	filesConfigurationsChan := make(chan *extractor.FileConfigurations, concurrency)
 	invalidFilesChan := make(chan *extractor.InvalidFile, concurrency)
+	pathsChan := make(chan string, concurrency)
+
+	go func() {
```

Comment:
For clarity, I suggest to wrap this in a private method maybe `parsePathsArrayToChan()`